docs/source/TQNet_Machine_Learning/layers/Double Hybrid Network (DHN) Layer.md

```diff
@@ -0,0 +1,87 @@
+# Double Hybrid Network (DHN) Layer
```

Comment:
how is this file supposed to be auto-used? You need to also link it in the actual rst file.